Dockerfile

```diff
@@ -67,4 +67,4 @@ ENTRYPOINT ["/tmpmount"]
 ADD . /go/src/github.com/opencontainers/runc
 
 RUN . tests/integration/multi-arch.bash \
-    && curl -o- -sSL `get_busybox` | tar xfJC - ${ROOTFS}
+    && curl -o- -sSL `get_busybox` | tar xfJC - ${ROOTFS}
```

Comment:
Looks like this inadvertently removed a newline

Reply:
I have removed them and learned more about git squashing ;)